test/ttt_test.exs

```diff
@@ -78,47 +78,47 @@ defmodule TTTTest do
   end
 
   test "Player tokens are updated and rendered on the board when user provides valid input" do
-    board = Board.setup
-    p1 = Player.human("Steven", :x)
-    p2 = Player.human("Connie", :o)
-    game = Game.setup(p1, p2)
-    {:ok, game} = Game.play_turn(game, :o, {0, 0})
-    {:ok, game} = Game.play_turn(game, :x, {2, 1})
-    {:ok, game} = Game.play_turn(game, :o, {1, 0})
-    {:ok, game} = Game.play_turn(game, :x, {1, 1})
-    {:ok, game} = Game.play_turn(game, :o, {2, 0})
+    board = Board.setup("", "")
+    p1 = Player.human({"Steven", "x"})
+    p2 = Player.human({"Connie", "o"})
+    game = Game.setup(p1, p2, 1)
+    {:ok, game} = Game.play_turn(game, p2, {0, 0})
+    {:ok, game} = Game.play_turn(game, p1, {2, 1})
+    {:ok, game} = Game.play_turn(game, p2, {1, 0})
+    {:ok, game} = Game.play_turn(game, p1, {1, 1})
+    {:ok, game} = Game.play_turn(game, p2, {2, 0})
```

Comment:
This is pretty verbose. Consider creating a helper function in the test file that will let you make multiple moves.